docs/docker-compose.yml

```diff
@@ -56,6 +56,7 @@ services:
       FEDERATION_QUEUE_USER: sublinks
       FEDERATION_QUEUE_PASS: sublinks
       FEDERATION_QUEUE_NAME: federation
+      FEDERATION_ROUTING_KEY: federation
```

Comment:
Routing key needs to be hard-coded and will vary based on object sent. See https://github.com/sublinks/sublinks-federation/pull/43/files#diff-058eae10080651784c26b74b93f5037b2d0d58736861057b3eb9c0b217eca334R51

Reply:
Whoops! Let me rename this var it should be the exchange and not the routing key